### ISSUE TITLE
Mixed ANCOVA

### ISSUE THREAD

**Issue by IndrajeetPatil** [jasp-stats/jasp-desktop#772]
Hi,

Is there a possibility to include mixed ANOVA with a covariate in Classical inference section of JASP? Currently, I can do ANCOVA or mixed ANOVA, but not mixed ANCOVA. 

Thanks.

**Comment by jonathon-love** [comment]
yup, we'll add in a covariates box soon.

do you want to handle this one @raviselker? we already do it for the Bayesian RM ANOVA.

technically, it should be RM ANCOVA, but i don't think we should get too hung up about the correct term.

**Comment by IndrajeetPatil** [comment]
Perfect! Thanks.

**Comment by raviselker** [comment]
Yes, shouldn't be too difficult to add this option.

**Comment by IndrajeetPatil** [comment]
Sorry for being so pesky, but will this feature be available in JASP 0.8?

**Comment by jonathon-love** [comment]
should be soon @IndrajeetPatil !

**Comment by raviselker** [comment]
Very soon actually: I just implemented it!

**Comment by IndrajeetPatil** [comment]
Fantastic! Thanks a lot! :)

**Comment by ZuccheroZsuzsi** [comment]
Hi! 

I just noticed this issue, as I was searching for a way to add covariates to a classical (frequentist) repeated-measures ANOVA in JASP. Between-subjects factors (nomial & categorical) can be added by default, but I can't find how to add (continuous) covariates to the rmANOVA. 
According to this issue it was solved last september (2015), but I have version 7.1, and there it is not available, as far as I can see.. 

Did I understand the solved issue wrongly? Is there a way to add the covariate box to classical rmANOVA (so: rmANCOVA) after all?

Thanks in advance!
